sys/cddl/compat/opensolaris/sys/vnode.h

```diff
@@ -268,7 +268,12 @@ vn_rename(char *from, char *to, enum uio_seg seg)
 
 	ASSERT(seg == UIO_SYSSPACE);
 
-	return (kern_renameat(curthread, AT_FDCWD, from, AT_FDCWD, to, seg));
+	return (kern_renameat(curthread, 
+			      AT_FDCWD, 
+			      cheri_ptr(from, strlen(from) + 1), 
+			      AT_FDCWD, 
+			      cheri_ptr(to, strlen(to) + 1), 
+			      seg));
```

Comment:
This would be closer to style(9):
```suggestion
	return (kern_renameat(curthread, AT_FDCWD, 
	    cheri_ptr(from, strlen(from) + 1), AT_FDCWD, 
	    cheri_ptr(to, strlen(to) + 1), seg));
```

Reply:
Thanks, done!